definitions-tests/definitions_test.go

```diff
@@ -0,0 +1,86 @@
+package definitions_tests
```

Comment:
Also, I think this directory is better suited in `/test/definition-test`

Reply:
moved to /test/definition-test/